docs/HTTPSRR.md

```diff
@@ -32,6 +32,13 @@ resolver.
 The data extracted from the HTTPS RR is stored in the in-memory DNS cache to
 be reused on subsequent uses of the same hostnames.
 
+## limitations
+
+We have decided to work on the HTTPS RR support by following what seems to be
+(widely) used, and simply wait with implementing the details of the record
+that do not seem to be deployed. HTTPS RR a DNS field with many odd corners
```

Comment:
:wave: spotted a small typo in a drive-by review:
```suggestion
that do not seem to be deployed. HTTPS RR is a DNS field with many odd corners
```

Reply:
Thanks!